plugins/modules/katello_content_view_version.py

```diff
@@ -22,71 +22,73 @@
 module: katello_content_view_version
 short_description: Create, remove or interact with a Katello Content View Version
 description:
-    - Publish, Promote or Remove a Katello Content View Version
+  - Publish, Promote or Remove a Katello Content View Version
 author: Sean O'Keeffe (@sean797)
 requirements:
-    - "nailgun"
+  - apypie
 notes:
-    - You cannot use this to remove a Contnet View Version from a Lifecycle environment, you should promote another version first.
-    - For idempotency you must specify either C(version) or C(current_lifecycle_environment).
+  - You cannot use this to remove a Content View Version from a Lifecycle environment, you should promote another version first.
+  - For idempotency you must specify either C(version) or C(current_lifecycle_environment).
 options:
-    server_url:
-        description:
-            - URL of Foreman server
-        required: true
-    username:
-        description:
-            - Username on Foreman server
-        required: true
-    password:
-        description:
-            - Password for user accessing Foreman server
-        required: true
-    verify_ssl:
-        description:
-            - Verify SSL of the Foreman server
-        default: true
-        type: bool
-    content_view:
-        description:
-            - Name of the content view
-        required: true
-    organization:
-        description:
-            - Organization that the content view is in
-        required: true
-    state:
-       description:
-            - Content View Version state
-       default: present
-       choices:
-           - absent
-           - present
-    version:
-        description:
-           - The content view version number (i.e. 1.0)
-    lifecycle_environments:
-        description:
-            - The lifecycle environments the Content View Version should be in.
-        default: Library
-    force_promote:
-        description:
-            - Force content view promotion and bypass lifecycle environment restriction
-        default: false
-        type: bool
-        aliases:
-          - force
-    force_yum_metadata_regeneration:
-        description:
-            - Force metadata regeneration when performing Publish and Promote tasks
-    synchronous:
-        description:
-            - Wait for the Publish or Promote task to complete if True. Immediately return if False.
-        default: true
-    current_lifecycle_environment:
-        description:
-            - The lifecycle environment that is already associated with the content view version
-            - Helpful for promoting a content view version
+  server_url:
```

Comment:
these all can go if you use the doc fragment (this module was not using the nailgun cement, so not the fragment already)

Reply:
Right. That's what i was not used to see. 8)